tests/test_client_functional.py

```diff
@@ -2103,11 +2103,11 @@ def dumps(obj):
 
 async def test_raise_for_status_per_request(aiohttp_client) -> None:
 
-    async def handler_redirect(request):
+    async def handle(request):
```

Comment:
This is unrelated. Let's not mess with other parts of the code. Also, if you want to refactor the name it should still be `handle_request`.

Reply:
it's `handle` in the rest of the file...

Reply:
Anyway, it's unrelated. Non-atomic changes are bad.

Reply:
done, will change in another pr.